src/main/src/test/java/org/geoserver/test/GeoServerSystemTestSupport.java

```diff
@@ -112,10 +112,12 @@
 import org.geoserver.security.password.GeoServerDigestPasswordEncoder;
 import org.geoserver.security.password.GeoServerPBEPasswordEncoder;
 import org.geoserver.security.password.GeoServerPlainTextPasswordEncoder;
+import org.geoserver.util.EntityResolverProvider;
 import org.geotools.data.DataUtilities;
 import org.geotools.data.simple.SimpleFeatureSource;
 import org.geotools.util.logging.Log4JLoggerFactory;
 import org.geotools.util.logging.Logging;
+import org.geotools.xml.PreventLocalEntityResolver;
```

Comment:
Is this needed? Or left over from an earlier version?

Reply:
Indeed leftover, removing it